dev_tools/packaging/set-dev-version.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+# Copyright 2018 The Cirq Developers
```

Comment:
You changed the year in setup.py instead of here.

Reply:
doh